lib/ansible/modules/network/panos/panos_object.py

```diff
@@ -31,7 +31,7 @@
 description:
     - Policy objects form the match criteria for policy rules and many other functions in PAN-OS. These may include
       address object, address groups, service objects, service groups, and tag.
-author: "Bob Hagen (@rnh556)"
+author:
```

Comment:
I don't think author can be empty
```
  File "../templates/plugin.rst.j2", line 433, in top-level template code
    {%   for author_name in author %}
TypeError: 'NoneType' object is not iterable
make: *** [modules] Error 1
```

Reply:
Update `templates/plugin.rst.j2`

Reply:
The fix should really make `author` and empty list, rather than test for a None value IMO.

Reply:
@dagwieers you mean updating the schema rather than `templates/plugin.rst.j2` ?

Reply:
No, I mean fixing plugin_formatter.py, rather than the template. I think it's good practice to keep templates as concise as possible and ensure the data-structures are clean and consistent. So I prefer making `author` always a list.